analytics/build.gradle

```diff
@@ -32,6 +32,7 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    namespace 'org.odk.collect.analytics'
```

Comment:
The new AGP prefers the package name declaration in `build.gradle` rather than `AndroidManifest.xml`. Definitely seems like nice change to me.